src/components/Sum.jsx

```diff
@@ -1 +1,15 @@
-// export function Sum() {}
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from 'react';
+
+export function Sum(props) {
```

Comment:
```suggestion
export function Sum({ a, b }) {
```
It is better to destructure a and b from props object to make your code a bit cleaner